### ISSUE TITLE
Possible bug in postgres query/data

### ISSUE THREAD

**Issue by wileyj** [hirosystems/stacks-blockchain-api#1047]
Not sure what's going on here, but using the rosetta endpoint for block `47700` results in a 5XX:
```bash
$ curl -X POST -H "Content-Type: application/json" \
    -d '{"network_identifier": {"blockchain": "stacks","network": "mainnet"},"block_identifier": {"hash": "0xfae4880e44341623b202d474cb4627ad9221361c9715389bfd5e43feba27704c"}}' \
    https://stacks-node-api.mainnet.stacks.co/rosetta/v1/block
{
  "error": "Error: FT metadata not found for token: SP343J7DNE122AVCSC4HEK4MF871PW470ZSXJ5K66.fractal-v1-7::fractal",
  "stack": "Error: FT metadata not found for token: SP343J7DNE122AVCSC4HEK4MF871PW470ZSXJ5K66.fractal-v1-7::fractal\n    at getValidatedFtMetadata (/app/src/rosetta-helpers.ts:1015:11)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at processEvents (/app/src/rosetta-helpers.ts:218:28)\n    at getOperations (/app/src/rosetta-helpers.ts:154:5)\n    at getRosettaBlockTransactionsFromDataStore (/app/src/api/controllers/db-controller.ts:518:24)\n    at getRosettaBlockFromDataStore (/app/src/api/controllers/db-controller.ts:329:16)\n    at /app/src/api/routes/rosetta/block.ts:33:21",
  "errorTag": "3fa5c6bf-8936-4f97-8041-8b81967e236b"
}%  
```

Compare with a request for block `47701`:
```bash
curl -X POST -H "Content-Type: application/json" \
    -d '{"network_identifier": {"blockchain": "stacks","network": "mainnet"},"block_identifier": {"hash": "0x8e1d85f8c76cf866c4f1f8fda3a79c55d854e7d117d76dbb551ca4497bc4421c"}}' \
    https://stacks-node-api.mainnet.stacks.co/rosetta/v1/block
{
  "block": {
    "block_identifier": {
      "index": 47701,
      "hash": "0x8e1d85f8c76cf866c4f1f8fda3a79c55d854e7d117d76dbb551ca4497bc4421c"
    },
    "parent_block_identifier": {
      "index": 47700,
      "hash": "0xfae4880e44341623b202d474cb4627ad9221361c9715389bfd5e43feba27704c"
    },
    "timestamp": 1644112976000,
    "transactions": [
      {
...
```

**Comment by wileyj** [comment]
@rafaelcr

**Comment by rafaelcr** [comment]
@wileyj it seems the `SP343J7DNE122AVCSC4HEK4MF871PW470ZSXJ5K66.fractal-v1-7` contract declares a Fungible Token which is not SIP-010 compatible. The API is trying to display transaction information (a Rosetta `operation`) which uses that token but it can't.

We have an open issue for a strict mode https://github.com/hirosystems/stacks-blockchain-api/issues/1007 (or other modes we could add) on FT/NFT metadata processing that would try to prevent these errors but we'd probably have to think about it the other way i.e. creating a mode that is more lax in a best-effort kind of way (only show operations for tokens that have metadata, ignore all others).

cc @zone117x

**Comment by rafaelcr** [comment]
@wileyj not at the moment, but I'll create a fix very shortly that will allow you to downgrade these errors to warnings so you can display these blocks even though they don't have complete FT data... I'll get back to you ASAP

**Comment by wileyj** [comment]
excellent thanks! i'll let the initial reporter know (doesn't affect me directly, was reported to me)

**Comment by rafaelcr** [comment]
@wileyj the fix is now released in master as v2.1.0
Let me know if I can help with anything else
